crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -518,9 +513,8 @@ impl ArtifactFetcher {
         for module in modules {
             if let Some(debug_id) = module.debug_id {
                 debug_ids.insert(debug_id);
-                // If we have a `DebugId`, we assume that the `DebugId`-based lookup will succeed.
-                // In that case, we do not want to look up the file by its file stem.
-            } else if let Ok(url) = module.abs_path.as_ref() {
+            }
+            if let Ok(url) = module.abs_path.as_ref() {
```

Comment:
I would not do this in `prefetch`, as `prefetch` should give us some kind of bundle to work with. we would then expect files to be available in that bundle.

Reply:
If we wont do this in the prefetch, then `abs_path` key is locked in a cache as `NotFound` for the rest of processing, as `artifact-lookup` endpoint was not able to query for it due to `url=` query param not being passed here.